src/main/resources/data/murder_mystery/lang/en_us.json

```diff
@@ -1,3 +1,51 @@
 {
-    "game.murder_mystery.murder_mystery": "Murder Mystery"
+    "game.murder_mystery.mansion": "Murder Mystery: Mansion",
+    "game.murder_mystery.metro": "Murder Mystery: Metro",
+    "game.murder_mystery.murder_mystery": "Murder Mystery",
+    "item.murder_mystery.coin": "Coin",
+    "item.murder_mystery.detective_bow": "Detective's Bow",
+    "item.murder_mystery.murderer_blade": "Murderer's Blade",
+    "map.murder_mystery.mansion": "Murder Mansion",
+    "map.murder_mystery.metro": "Murder Metro",
+    "role.murder_mystery.detective": "Detective",
+    "role.murder_mystery.innocent": "Innocent",
+    "role.murder_mystery.murderer": "Murderer",
+    "text.murder_mystery.dead": "Dead",
+    "text.murder_mystery.death_quote": "\"%s\"",
+    "text.murder_mystery.death_quote.1": "Ouchie",
+    "text.murder_mystery.death_quote.10": "I'm looking for loopholes",
+    "text.murder_mystery.death_quote.11": "Gun's not loaded... see?",
+    "text.murder_mystery.death_quote.12": "No.",
+    "text.murder_mystery.death_quote.13": "Now why did I do that?",
+    "text.murder_mystery.death_quote.14": "Don't let it end like this. Tell them I said something important!",
+    "text.murder_mystery.death_quote.15": "Haha... fool.",
+    "text.murder_mystery.death_quote.16": "The tables seem to have turned...",
+    "text.murder_mystery.death_quote.17": "Hey! I saw this one!",
+    "text.murder_mystery.death_quote.18": "What are ya gonna do? stab me?",
+    "text.murder_mystery.death_quote.19": "Hey... you guys wanna see a dead body?",
+    "text.murder_mystery.death_quote.2": "Skibbity bop mm dada!",
+    "text.murder_mystery.death_quote.20": "10/10, would live again.",
+    "text.murder_mystery.death_quote.21": "Wow.",
+    "text.murder_mystery.death_quote.22": "It's beautiful, it's perfect, oh wow... just kidding.",
+    "text.murder_mystery.death_quote.3": "Oof",
+    "text.murder_mystery.death_quote.4": "Hey fellas!",
+    "text.murder_mystery.death_quote.5": "I did not get my Spaghetti-O's; I got spaghetti!",
```

Comment:
Any reason this ordering is weird?

Reply:
Alphabetical JSON sortering did that of its own, did not notice it